app/src/pages/ProtocolDashboard/ProtocolCard.tsx

```diff
@@ -91,6 +182,55 @@ export function ProtocolCard(props: {
             setShowDeleteConfirmationModal={setShowDeleteConfirmationModal}
           />
         )}
+        {showFailedAnalysisModal && (
+          <Modal
+            header={failedAnalysisHeader}
+            onOutsideClick={() => setShowFailedAnalysisModal(false)}
+          >
+            <Flex
+              flexDirection={DIRECTION_COLUMN}
+              gridGap={SPACING.spacing32}
+              width="100%"
+            >
+              <Flex
+                flexDirection={DIRECTION_COLUMN}
+                gridGap={SPACING.spacing8}
+                maxWidth="100%"
+              >
+                <Trans
+                  t={t}
+                  i18nKey={t('error_analyzing', { protocolName })}
+                  components={{
+                    block: (
+                      <StyledText
+                        as="p"
+                        css={css`
+                          display: -webkit-box;
+                          -webkit-box-orient: vertical;
+                          -webkit-line-clamp: 4;
```

Comment:
https://www.figma.com/file/AoTLAYuWawlaWItB1umOjr?node-id=8342:130756#476739265

```suggestion
                          -webkit-line-clamp: 3;
```

Reply:
oooh, i guess i was confused because doesn't 1 line count for the other text in the string? So 4 lines technically means 3 lines for the protocol name?

Reply:
I think this case is a little bit tricky since sometimes we need +1 lines or the same lines.

now looks good.
![Screenshot 2023-06-21 at 12 18 11 PM](https://github.com/Opentrons/opentrons/assets/474225/25aa5f79-5687-461d-bc26-a01e9f178852)